_data/index/projects.yml

```diff
@@ -17,7 +17,7 @@
 
 - name: Problem of the Week
   i18n: google
-  url: https://docs.google.com/document/d/1J2c5zkhvZQ8U2S5PNxZoTQce0MO2JCdPOrnfViZ2Lv8/edit?usp=sharing
+  url:  /resources/potw.html
```

Comment:
Also, check the link using in Navbar. We don't use 'html' suffix in links.

Reply:
Made changes according to both the above reviews